te/GET_AND_RUN_DOCKER_IMAGE.py

```diff
@@ -92,7 +92,7 @@ def getLogger(self):
 class DOCKER_INITIALIZER:
 
     def __init__(self, pathToDocker, ip, nginxPort, stat_collect_interval, stat_dump_interval, \
-        basePath, typeOfDocker, te_controller_ip, te_logpath, te_loglevel):
+        basePath, typeOfDocker, te_controller_ip, te_logpath, te_loglevel, my_ip, flask_port):
```

Comment:
GET_AND_RUN_DOCKER_IMAGE.py takes the flask ip and its mgmt_ip as arg if it is called for setup_tedp() -- there by can start the RQ_CONNECTOR.py process